README.md

```diff
@@ -171,10 +171,11 @@ For further details, consult the [wiki](https://github.com/ReactiveX/RxJava/wiki
 
 ## Communication
 
+- [Gitter.im](https://gitter.im/ReactiveX/RxJava)
```

Comment:
Realtime -> "Slightly Slower" communication -> Social Media was my idea. 
Unless you want it sorted by preferred communication channels, then i'll update it to your liking.